src/PipeScore/global/settings.ts

```diff
@@ -56,6 +58,7 @@ export class Settings {
       bpm: this.bpm,
       gapAfterGracenote: this.gapAfterGracenote,
       harmonyVolume: this.harmonyVolume,
+      instrument: this.instrument,
```

Comment:
Once `this.instrument` becomes an enum, we'll need a way of converting to/from string, and use that here, and in `fromJSON`, and also when we're getting the audio file paths.